tests/lib/account.js

```diff
@@ -183,7 +260,7 @@ define([
         var account;
         var opts = {
           service: 'sync',
-          redirectTo: 'https://sync.firefox.com/after_reset',
+          redirectTo: 'https://sync.127.0.0.1/after_reset',
```

Comment:
The `redirectTo` was updated because of https://github.com/mozilla/fxa-auth-server/pull/2403. I opted to updated here instead of specifing `REDIRECT_DOMAIN=firefox.com` when the server is started this mimics what value fxa-local-dev would have in the scenario you wanted to test against that auth-server.

Reply:
@vbudhram how come there is no port here? is that a problem?

Reply:
I don't believe so? The redirects in these cases appear to be made up and not used. This does pass the validation though.

Reply:
right, okay I guess we should be okay until we have to use it